tools/tests/interaction-manager/draggable/test.js

```diff
@@ -0,0 +1,198 @@
+const createDraggableElement = async () => {
+    const container = document.createElement('DIV');
+
+    container.style.width = '500px';
+    container.style.height = '500px';
+    container.classList.add('container');
+
+    const template = `<div class="square" style="background-color: cadetblue; width: 200px; height: 200px;"></div>`;
+    container.innerHTML = template;
+
+    document.body.appendChild(container);
+
+    await waitDraggableFrames();
+};
+
+const waitDraggableFrames = () => {
+    return new Promise((resolve) => {
+        waitForStyles(resolve, 4);
+    });
+};
+
+const removeDraggableElement = () => {
+    const container = document.querySelector('.container');
+
+    document.body.removeChild(container);
+};
+
+const dragElement = (square, x, y) => {
+    const squareElement = square.draggableElements[0];
+    square.onMouseDown({ currentTarget: squareElement });
+    square.onMouseMove({ clientX: x, clientY: y });
+    square.onMouseUp();
+};

```

Comment:
Maybe make util for this? I saw that you are using it a couple of times with different parameters.

Reply:
Added the util in actions.js as `dragIMElement()`